autosubmit/job/job_list.py

```diff
@@ -1134,18 +1129,16 @@ def _calculate_filter_dependencies(self, filters_to_apply, dic_jobs, job, depend
         :param chunk: Chunk
         :param graph: Graph
         :param dependencies_keys_without_special_chars: Dependencies keys without special chars
-        :param distances_of_current_section: Distances of current section
-        :param distances_of_current_section_members: Distances of current section members
-        :param key: Key
         :param dependencies_of_that_section: Dependencies of that section
         :param chunk_list: Chunk list
         :param date_list: Date list
         :param member_list: Member list
         :param special_dependencies: Special dependencies
         :param problematic_dependencies: Problematic dependencies
-        :return:
 
+        :return:
```

Comment:
AFAIK there's no need to separate return statements (I think in Java that was a thing?). 

e.g. https://github.com/python/cpython/blob/08d7687094c6acb8c2ea1925a292a94ce1246c82/Lib/email/quoprimime.py#L84-L94